.github/CODEOWNERS

```diff
@@ -6,36 +6,23 @@
 * @asaj @cmcewen @timmoreton
 
 # directory and file-level owners. Feel free to add to this!
+/packages/attestation-proxy/ @celo-org/cap
+/packages/attestation-service/ @celo-org/cap
 /packages/azure/ @codyborn
-/packages/blockchain-api/ @jmrossy @cmcewen @i1skn @jeanregisser
 /packages/celotool/ @asaj
-/packages/cli/ @mcortesi @gastonponti @medhak1 @celo-org/devx
-/packages/sdk/ @mcortesi @gastonponti @medhak1 @barbaraliau @celo-org/devx
-/packages/dappkit/ @cmcewen @barbaraliau @celo-org/devx
-/packages/dev-utils/ @mcortesi
+/packages/cli/ @celo-org/applications
+/packages/dev-utils/ @celo-org/applications
 /packages/docs/ @timmoreton @barbaraliau @critesjosh 
```

Comment:
should this be updated too?

Reply:
I think this is supposed to go to @celo-org/devrel

Reply:
Remove